email_templates/cashEmail/html.handlebars

```diff
@@ -0,0 +1,7 @@
+<h1>{{listing_name}} has been purchased!</h1>
+
+IOU: By purchasing {{listing_name}}, {{listing_buyer}} has agreed to pay the 
+amount of ${{listing_price}} in cash to {{listing_creator}}.
+
+Please save this email for your records until the IOU has been resolved. 
+Thank you for using Obay!
```

Comment:
Looks pretty good! Btw, you can write any html here as you normally would if you want to prettify things/make an email look real professional (ex could throw in the obay logo and the like).